Translators/WZDx/main.py

```diff
@@ -32,9 +40,9 @@ def update_rsu_region_name(request, tim_body):
 
 def translate(wzdx_geojson):
     tims = []
-    duration = os.getenv("DURATION_TIME", 1800)
+    duration = os.getenv("DURATION_TIME", 30)
     # if no RSUs found, drop that one
-    for feature in wzdx_geojson["features"]:
+    for feature in wzdx_geojson:

```

Comment:
Did we drop the "features" wrapper, or just not need to explicitly name it?

Reply:
Looks like our calling method now get's the "features" object, but we'll need to update that in the `translateWzdxTIM` call as well

Reply:
Updated translateWzdxTIM to just pass the "features" object to the translate method